kbs/src/admin/password.rs

```diff
@@ -0,0 +1,176 @@
+// Copyright (c) 2025 by NVIDIA.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use actix_web::{http::header::Header, HttpRequest};
+use actix_web_httpauth::headers::authorization::{Authorization, Bearer};
+use argon2::{Argon2, PasswordHash, PasswordVerifier};
+use jwt_simple::{
+    claims::NoCustomClaims,
+    common::VerificationOptions,
+    prelude::{Claims, Ed25519KeyPair, EdDSAKeyPairLike, EdDSAPublicKeyLike},
+};
+use log::info;
+use serde::Deserialize;
+use std::collections::HashMap;
+use std::path::PathBuf;
+
+use crate::admin::error::*;
+use crate::admin::AdminBackend;
+
+const ADMIN_TOKEN_ISSUER: &str = "Trustee Admin";
+
+#[derive(Clone, Debug, Default, Deserialize, PartialEq)]
+#[serde(default)]
+pub struct PasswordAdminConfig {
+    pub personas: Vec<PasswordPersona>,
+    /// The number of hours that an admin token will be valid for.
+    /// After the token expires, an admin must login again and receive
+    /// a new token.
+    pub admin_token_life_hours: u64,
+    /// The key pair used for signing and validating the admin tokens.
+    /// If a path is provided, the key pair will be loaded from a file.
+    /// This be used to share auth tokens between Trustee instances and
+    /// to allow Trustee to restart with invalidating existing tokens.
+    /// The key pair should be in PEM format.
+    /// If no path is provided, a random key pair will be generated.
+    pub key_pair_path: Option<PathBuf>,
```

Comment:
If this field is not used, let's delete it. Also, if we want to use the key pair, we can set pri and pub key separately with path.